draft-ietf-ppm-dap.md

```diff
@@ -1170,10 +1164,17 @@ struct {
   };
 } PartialBatchSelector;
 
+struct {
+  ReportMetadata metadata;
+  opaque public_share<0..2^32-1>;
+  opaque leader_prep_share<0..2^32-1>;
+  HpkeCiphertext encrypted_input_share;
```

Comment:
* I think the notion of "report share" remains useful even after this change. I think it would be useful to keep the struct around.
* I'd have to double check, but are we bumping the length tag for the prep share from 16 to 32 bits in this change? 16 ought to be sufficient.

```suggestion
  ReportShare report_share;
  opaque leader_prep_share<0..2^32-1>;
```

Reply:
> Hmm -- ReportShare was replaced (renamed & modified) with ReportAggregationInitReq, for consistency with ReportAggregation{Init,Continue}{Req,Resp} naming.

This PR is quite large; to minimize reviewer effort, can we try to limit it to essential changes only?

Reply:
In particular, in general I think we should try to avoid renaming structures, even if their contents change. In this particular case I don't think we need change the meaning (or even the contents of) ReportShare. The report share can still just be the encrypte dinput share and meta data; then the prep share could be a separate structure.